packages/cloud2edge/requirements.yaml

```diff
@@ -12,8 +12,8 @@
 #
 dependencies:
   - name: hono
-    version: ^1.10.18
+    version: ~2.0.7
     repository: "https://eclipse.org/packages/charts/"
   - name: ditto
-    version: ~2.4.0
+    version: ~2.5.5
```

Comment:
It would be good to increase this to 2.5.6, including the fix for #398.

Reply:
done